kegtab/src/main/java/org/kegbot/app/TapStatusFragment.java

```diff
@@ -256,6 +258,21 @@ public void onClick(View v) {
       description = tap.getDescription();
     }
 
+    // Find ABV and IBU values
+    final String abv = String.valueOf(keg.getBeverage().getAbvPercent());
+    if(mCore.getConfiguration().getAbvVisibleWhenZero()) {
+      abvText.setText("ABV: " + abv + "%");
+    } else{
+      abvText.setText(keg.getBeverage().getAbvPercent() == 0 ? "" : "ABV: " + abv + "%");
+    }
+
+    final String ibu = String.valueOf(Math.round(keg.getBeverage().getIbu()));
+    if(mCore.getConfiguration().getIbuVisibleWhenZero()){
+      ibuText.setText("IBU: " + ibu);
+    } else{
+      ibuText.setText(keg.getBeverage().getIbu() == 0 ? "" : "IBU: " + ibu);
```

Comment:
This should probably just set the visibility to GONE. `ibuText.setVisibility(View.GONE);`

Reply:
Agreed, fixing